test/tests/benchmark/utils/time_logger.py

```diff
@@ -0,0 +1,61 @@
+import os
+import time
+import json
+
+"""
+Class to manage time logger file
+"""
+
+class timeLogger(object):
```

Comment:
This seems like common functionality; although being introduced for benchmarking it could potentially be used in other places?  Either way, I would move this into /modules/logger.py and look at leveraging the Python Logging library handler objects: https://docs.python.org/2/library/logging.html#logging.Logger.addHandler.